sources/include/parser/cityobjectelementparser.h

```diff
@@ -55,6 +55,10 @@ namespace citygml {
         void parseGeometryForLod0FootPrintNode();
         void parseImplicitGeometryForLODLevel(int lod);
         void parseGeometryPropertyElementForLODLevel(int lod, const std::string& id);
+
+        std::string _key_codeSpace;
+        std::string _codeValue_codeSpace;
+        std::string _attributeKey;
```

Comment:
命名規則をlibcitygmlのほかの部分と統一お願いいたします。
また、`_attributeKey`については`m_lastAttributeName`で代用お願いいたします。
```suggestion
        std::string m_lastCodeSpace;
        std::string m_lastCode;
```